azure-pipelines/ui-automation/msal-broker-release.yml

```diff
@@ -0,0 +1,147 @@
+# run MSAL with Broker UI automation testcases
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'gCloudProjectId' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+name: $(Build.BuildId)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "0 5 * * 1-6" # 5:00 AM UTC everyday Mon-Sat
+    displayName: Daily Local MSAL UI Automation with Prod Broker Run
+    branches:
+      include:
+        - dev
+    always: true
+
+parameters:
+  - name: firebaseDeviceId
+    type: string
+    displayName: Firebase Device Id
+    default: blueline
+    values:
+      - blueline
+      - flame
+      - redfin
+      - sailfish
+      - walleye
+      - bluejay
+      - oriole
+  - name: firebaseDeviceAndroidVersion
+    displayName: Firebase Device Android Version
+    type: number
+    default: 28
+    values:
+      - 25
+      - 26
+      - 27
+      - 28
+      - 29
+      - 30
+      - 31
+      - 32
+      - 33
+  - name: testTargetPackages
+    displayName: Packages as Test Targets
+    type: string
+    default: package com.microsoft.identity.client.msal.automationapp.testpass.broker
```

Comment:
Could add: notAnnotation org.junit.Ignore. This will cause higher pass rate percentage since ignored tests determent the pass rate in ado.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1690/commits/2b3b116cd0c1b3e073c995854ea1783748954f32